lib/modules/manager/helmfile/extract.spec.ts

```diff
@@ -202,5 +202,82 @@ describe('modules/manager/helmfile/extract', () => {
         ],
       });
     });
+
+    it('parses a chart with a go templating', () => {
+      const content = `
+      repositories:
+        - name: kiwigrid
+          url: https://kiwigrid.github.io
+      releases:
+        - name: example
+      {{- if neq .Values.example.version  "" }}
+          version: {{ .Values.example.version }}
+      {{- else }}
+          version: 1.0.0
+      {{- end }}
+          chart: stable/example
+        - name: example-internal
+          version: 1.0.0
+          chart: stable/example
+      `;
+      const fileName = 'helmfile.yaml';
+      const result = extractPackageFile(content, fileName, {
+        aliases: {
+          stable: 'https://charts.helm.sh/stable',
+        },
+      });
+      expect(result).toMatchSnapshot({
```

Comment:
please use `toMatchObject`

Reply:
Updated in 38079014c9b0f00d101091228509bb8796df26ad

Reply:
Please read https://github.com/renovatebot/renovate/blob/main/.github/contributing.md#resolve-review-comments-instead-of-commenting and reduce noise 🤗